concert_service_manager/src/concert_service_manager/service_instance.py

```diff
@@ -33,31 +34,33 @@ def dummy_cb():
 class ServiceInstance(object):
 
     __slots__ = [
-            'msg',                  # concert_msgs.ServiceProfile fixed and variable parameters
-            '_update_callback',     # used to trigger an external callback (service manager publisher) when the state changes.
-            '_namespace',           # namespace that the service will run in
-            '_lock',                # protect service enabling/disabling
-            '_proc',                # holds the custom subprocess variable if TYPE_CUSTOM
-            '_roslaunch',           # holds the roslaunch parent variable if TYPE_ROSLAUNCH
-            '_shutdown_publisher',  # used for disabling the service
-            # aliases
-            'name',
-        ]
+        'msg',                  # concert_msgs.ServiceProfile fixed and variable parameters
+        
+        '_namespace',           # namespace that the service will run in
+        '_lock',                # protect service enabling/disabling
+        '_proc',                # holds the custom subprocess variable if TYPE_CUSTOM
+        '_roslaunch',           # holds the roslaunch parent variable if TYPE_ROSLAUNCH
+        '_shutdown_publisher',  # used for disabling the service
+        '_concert_name',  # todo
+        # aliases
+        'name',
+    ]
 
     shutdown_timeout = 5
     kill_timeout = 10
 
-    def __init__(self, service_profile=None, env=os.environ, update_callback=dummy_cb):
+    def __init__(self, concert_name=None, service_profile=None, env=os.environ):
         '''
           @param service_profile :
           @type concert_msgs.msg.ConcertService
         '''
+        self._concert_name = concert_name.lower().replace(' ', '_')
```

Comment:
wrapping name seems happenning in many places. How about create a method in `rocon_python_utils` to wrap names.
e.g) rocon_python_utils.ros.get_ros_friendly_name(concert_name).